guides.md

```diff
@@ -335,7 +335,7 @@ Prior to creating an invoice, confirm with the client whether they would like to
 
 ### Settling an invoice payment
 
-Once the client has paid the invoice, we must check the amount received at our receiving account at [Desjardins](https://www.desjardins.com/ca/) (domestic) or [TransferWise](https://wise.com) (foreign currency), and record the transaction in [Quickbooks Online](https://quickbooks.intuit.com/ca/).
```

Comment:
Transfer wise does not like being crawled by github throwing error 504. This workaround is used in other places to skip the crawl